src/modules/ie-objects/services/ie-objects.service.ts

```diff
@@ -275,6 +284,23 @@ export class IeObjectsService {
 		return allAdapted;
 	}
 
+	public async findObjectsForSitemap(licenses: IeObjectLicense[]): Promise<IeObjectsSitemap[]> {
+		try {
+			const { object_ie: objects } = await this.dataService.execute<
```

Comment:
REQ: rename objects to u ieObjects. This to adhere to naming conventions and avoid confusion with Javascripts own object.